### PR TITLE
Fixes #676 : Show all info about a season on season's “show” page

### LINKED ISSUE #676: Show all info about a season on season's “show” page
Currently, the season's show view looks like this:
screen shot 2017-02-27 at 10 42 32

In order to view all information about a season's application opening and closing dates, we need to go into the edit view (see below), which is unpractical. 

screen shot 2017-02-27 at 10 36 09

I'd suggest having all important season dates in the season “show” view directly.

This should be a fairly quick fix, even for someone not extremely familiar with the Teams App architecture. :)

### PR DESCRIPTION
**Pull request by amrtanair**
Added the important season dates like application open date in the season's “show” view directly.

### CODE REVIEW

File: app/views/orga/seasons/_form.html.slim~ [hunk 1]
```diff
@@ -0,0 +1,16 @@
+= simple_form_for([:orga, @season]) do |f|
```
Comment (carpodaster) [inline app/views/orga/seasons/_form.html.slim~]: This seems like a backup file, automatically created by your editor. Can you please remove this from the PR again? Hint: it's always best to select/stage the files for a commit individually; avoid using `git add .` which will also add unwanted backup files like this one.

File: app/views/orga/seasons/show.html.slim~ [hunk 2]
```diff
@@ -0,0 +1,18 @@
+= render 'orga/breadcrumbs'
```
Comment (carpodaster) [inline app/views/orga/seasons/show.html.slim~]: This seems like a backup file, automatically created by your editor. Can you please remove this from the PR again? Hint: it's always best to select/stage the files for a commit individually; avoid using `git add .` which will also add unwanted backup files like this one.

File: app/views/orga/seasons/show.html.slim [hunk 3]
```diff
@@ -10,6 +10,12 @@ h1 Season #{@season.name}
     dd = @season.starts_at
     dt End Date
     dd = @season.ends_at
+    dt Applications open at
+    dd = @season.applications_open_at
+    dt Applications close at
+    dd = @season.applications_close_at
+
+
```
Comment (carpodaster) [inline app/views/orga/seasons/show.html.slim]: There is also `project_proposals_open_at` and `project_proposals_close_at`. Could you please add those, too? Thank you!

File: app/views/orga/seasons/show.html.slim [hunk 4]
```diff
@@ -10,6 +10,12 @@ h1 Season #{@season.name}
     dd = @season.starts_at
     dt End Date
     dd = @season.ends_at
+    dt Applications open at
+    dd = @season.applications_open_at
+    dt Applications close at
+    dd = @season.applications_close_at
+
```
Comment (carpodaster) [inline app/views/orga/seasons/show.html.slim]: (Additional empty lines – can you please remove them?)

**Comment by carpodaster** [comment]
> 👍 for changing the prepositions to 'on' :-)

All the fields are date *time* fields. While I am not a native speaker, I believe »at« is more appropriate for a concrete time. I don't have strong feelings about it, though. Let's merge this and maybe change the preposition later. :shipit:

**Comment by carpodaster** [comment]
Thanks @amrtanair !

**Comment by alicetragedy** [comment]
:partyparrot:

**Pull request merged**